src/admin/admin.service.ts

```diff
@@ -155,4 +155,49 @@ export class AdminService {
       };
     }
   }
+
+  async addStudents(students: StudentToImport[]) {
```

Comment:
a co to dokładnie robi?

Reply:
Wrzucam w insomni jsona z tablicą danych o studentImport i mi się dodają. Nie wiedziałem jak zasymulować wrzucenie pliku i nie chciało mi się wpisywać ich do heidi ręcznie.

Reply:
Skoro mamy już gotowy endpoint do tego...
W insomni wpisujesz endpoint i wybierasz jako typ danych zamiast JSON --> Multipart. Tam wpisujesz nazwe pliku i wybierasz plik z dysku (wrzuciłem przykładowy plik do projektu).
<img width="541" alt="image" src="https://user-images.githubusercontent.com/61238271/182599083-06aa68e8-40b9-41ef-96f8-da1271e68d47.png">
<img width="392" alt="image" src="https://user-images.githubusercontent.com/61238271/182599213-82c8c259-fe86-446f-99e5-5202ccd81f3c.png">